### ISSUE TITLE
Aedict Reliable Kanji Search Crash

### ISSUE THREAD

**Issue by cntradction** [mvysny/aedict#872]
1. Google Pixel 2
2. Android Q Beta
3. https://www.youtube.com/watch?v=   N7fydlgWrf0
4. Aedict Kanji Search reliably crashes whenever 25 or more strokes are written in.

**Comment by mvysny** [comment]
Thank you! I have a crash in my console, I'll try to look at this ASAP. Crash details:
```
Caused by java.lang.IllegalStateException: invalid start/toIndex: 21/19 blocksize=3 threadCount=8 list.size()=19
       at com.leafdigital.kanji.KanjiList.getTopMatches(KanjiList.java:242)
       at sk.baka.aedictkanjipadext.Searcher$MyLoader.load(Searcher.java:159)
       at sk.baka.aedictkanjipadext.Searcher$MyLoader.loadItemsInBackground(Searcher.java:141)
```

**Comment by mvysny** [comment]
Fixed in 3.50.14

**Comment by cntradction** [comment]
Wow, that was fast! Thank you so much!
